inst/templates/ad_advs.R

```diff
@@ -0,0 +1,271 @@
+# Name: ADVS
+#
+# Label: Vital Signs Analysis dataset
+#
+# Input: adsl, vs
+
+
+# Attach/load required packages ----
+library(admiral)
+library(admiralmetabolic)
+library(tibble)
+library(dplyr)
+library(stringr)
+
+
+# Define project options/variables ----
+# Use the admiral option functionality to store subject key variables in one
+# place
+set_admiral_options(subject_keys = exprs(STUDYID, USUBJID))
+
+# Store ADSL variables required for derivations as an R object, enabling
+# simplified usage throughout the program
+adsl_vars <- exprs(TRTSDT, TRTEDT, TRT01P, TRT01A)
+
+
+# Read in data ----
+# See the "Read in Data" vignette section for more information:
+# (https://pharmaverse.github.io/admiral/articles/bds_finding.html#readdata)
+
+# Read data
+vs_metabolic <- admiralmetabolic::vs_metabolic
```

Comment:
```suggestion
vs_metabolic <- admiralmetabolic::vs_metabolic %>%
  filter(
    !(VSSTAT %in% "NOT DONE")
  )
  
```

Reply:
@yurovska There is no VSSTAT == "NOT DONE" in the current vs data and thus this filter was removed from both vignette and template to avoid confusion to the user as to why this filter was added.

Reply:
I know there are no cases like that but thought it would be nice to do that just in case the vs data changes in the future. I'm not insisting though.

Reply:
We actually removed this earlier and also removed it from the vignette. But I think we might want to revisit this later, as it could make sense to have this (both here and in the vignette), when its in our dataset. We should perhaps consider adding some records to the data as well.

Reply:
No objections to keeping it out for now and revising later.